deploy/examples/create-external-cluster-resources.py

```diff
@@ -25,11 +25,12 @@
 from os import linesep as LINESEP
 from os import path
 from email.utils import formatdate
-import urllib.parse
-
 import requests
 from requests.auth import AuthBase
 
+if sys.version_info.major >= 3:
```

Comment:
you can remove this and also line #54 to #59 

And then try running the CI

Reply:
this is not available in python2 so that's why I added this check

Reply:
thanks for the comment I got the fix.